src/main/java/ac/grim/grimac/checks/Check.java

```diff
@@ -60,6 +61,13 @@ public boolean shouldModifyPackets() {
         return isEnabled && !player.disableGrim && !player.noModifyPacketPermission && !exempted;
     }
 
+    public void updateExempted () {
```

Comment:
I added a space here for some reason, oops
```suggestion
    public void updateExempted() {
```